### PR TITLE
modify sync log description about lower block

### PR DESCRIPTION
**Pull request by LucasLi1024**

### CODE REVIEW

File: bcos-sync/bcos-sync/BlockSync.cpp [hunk 1]
```diff
@@ -215,8 +215,8 @@ void BlockSync::printBehindPeers()
                 BLKSYNC_LOG(ERROR) << "The lowest blocknumber is too far behind the highest block "
                                       "number queried by getSyncStatus, "
                                    << "[highest block number: " << knownHighestNumber
-                                   << ", current nodeid: " << _p->nodeId()->shortHex()
-                                   << ", current block number: " << _p->number() << "]";
+                                   << ", lower nodeid: " << _p->nodeId()->shortHex()
```
Comment (bxq2011hust) [inline bcos-sync/bcos-sync/BlockSync.cpp:218]:
```suggestion
                                   << ", lower node: " << _p->nodeId()->shortHex()
```

File: bcos-sync/bcos-sync/BlockSync.cpp [hunk 2]
```diff
@@ -215,8 +215,8 @@ void BlockSync::printBehindPeers()
...
+                                   << ", lower block number: " << _p->number() << "]";
```
Comment (bxq2011hust) [inline bcos-sync/bcos-sync/BlockSync.cpp:219]:
```suggestion
                                   << ", lower node block number: " << _p->number() << "]";
```

File: bcos-sync/bcos-sync/BlockSync.cpp (hunk 1 above)
Comment (LucasLi1024) [reply bcos-sync/bcos-sync/BlockSync.cpp]: fixed

File: bcos-sync/bcos-sync/BlockSync.cpp (hunk 2 above)
Comment (LucasLi1024) [reply bcos-sync/bcos-sync/BlockSync.cpp]: fixed

**Review by bxq2011hust — APPROVED** [review]

**Review by JimmyShi22 — APPROVED** [review]

**Pull request merged**